spec.bs

```diff
@@ -684,6 +689,12 @@ input {{SecurePaymentConfirmationRequest}} |data|, are:
   authentication-rejected.https.html
 </wpt>
 
+1. If the [=system visibility state=] of the [=top-level browsing context's=]
+     [=traversable navigable=] is not `"visible"`, return `false`.
```

Comment:
> Is there a reason I'm missing that you picked to go through the traversable navigable instead of looking at the Document's visibility state? 

Not really, other than I had this vague notion that one was meant to use navigables these days instead of 'something'. But that 'something' may not be the Document, I just couldn't recall!

Your change seems good to me, with one question - does using the direct `[=Document/visibility state=]` behave sensibly in the case of the API being called from an iframe? (I.e., does it always correspond to the visibility of the top-level window/tab that is either visible or not, and is it updated in a way that developers would find intuitive/sensible?)

Reply:
Thanks, I've made the change now.

> The 'something' is "browsing contexts", fwiw. :)

Ah yes, now I remember 🤣